File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.cpp
#include <iostream>

using namespace std;

int main(){
    int l;
    double valor, soma=0;
    char escolha;
    cin >> l;
    cin >> escolha;
    for(int i=0;i<12;i++){
        for(int j=0;j<12;j++){
            cin >> valor;
            if(j==l){
                soma+=valor;
            }
        }
    }
    cout.precision(1);
    cout.setf(ios::fixed);
    if(escolha=='S'){
        cout << soma << endl;
    } else{
        cout << soma/12 << endl;
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.rb
soma=0
l = gets.to_i
escolha = gets.to_s
for i in 0..11 do
    for j in 0..11 do
        valor = gets.to_f
        if(j==l) then
            soma=soma+valor
        end
    end
end
    
if(escolha=="S") then
    puts "%0.01f" % soma.round(1)
else
    puts "%0.01f" % (soma/12).round(1)
end